README.md

```diff
@@ -2,92 +2,133 @@
+## Architecture
 
+The Portal Network is built upon on the [Discover V5 protocol](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md) and operates over the UDP transport.
 
-### Scalable Lightweight Clients
+The Discovery v5 protocol allows building custom sub-protocols via the use of the built in TALKREQ and TALKRESP message.
 
-The term "light client" tends to refer to a client of the existing DevP2P LES network.  This network is designed using a client/server architecture.  The LES network has a total capacity dictated by the number of "servers" on the network.  In order for this network to scale, the "server" capacity has to increase.  This also means that at any point in time the network has some total capacity which if exceeded will cause service degredation across the network.  Because of this the LES network is unreliable when operating near capacity.
+The Portal Network is divided into the following sub-protocols.
 
-The Portal Network aims to solve this problem by designing our networks so that each additional client that joints the network adds additional capacity to the network.  The end result *should* be a network which becomes more robust and powerful as more nodes join the network.
+- State Network
+- History Network
+- Transaction Gossip Network
+- Header Gossip Network
+- Canonical Indices Network
 
-> Additional reading: https://snakecharmers.ethereum.org/the-winding-road-to-functional-light-clients/
-> 
-> Additional watching: https://www.youtube.com/watch?v=MZxqRs_tLNs
+Each of these networks is designed to deliver a specific unit of functionality.  Most portal clients will participate in all of these networks in order to deliver the full JSON-RPC API.  Each network however is designed to be independent of the others, allowing clients the option of only participating in a subset of them if they wish.
 
+All of the sub-protocols in the Portal Network establish their own Kademlia DHT network that is managed independent of the base Discovery V5 protocol Kademlia DHT.
 
-### JSON-RPC API
+
+## Design Principles
+
+Each of the Portal Network sub-protocols follows these design principles.
+
+1. Isolation
+  - Each network should be able to be run independent of the other networks.  Participation in one network should not require participation in another network.
+2. Distribution of Responsibility
+  - Normal operation of the network should result in a roughly even spread of responsibility across the individual nodes in the network.
+3. Tunable Resource Requirements
+  - Individual nodes should be able to control the amount of machine resources (HDD/CPU/RAM) they must expend to be part of the network.
```

Comment:
maybe: "Individual nodes should be able to control the amount of machine resources (HDD/CPU/RAM) they provide to the network"

Also, (2) and (3) seem to conflict with each other. I think (2) means to say nodes which have donated the same amount of disk space to the network are given roughly equal load (although I don't think that's true, whoever is close to uniswap will receive a lot of requests). Or, maybe (2) mean demands are made from nodes in proportion to how much they have offered. I'm not sure of a succinct way to express that.

Reply:
I like the cleanup on phrasing for (3).  I agree that (2) misses the mark a little but I'm also unsure how to cleanup the phrasing.  Gonna just leave it and hope to clean it up in a future pass if it remains confusing.